### PR TITLE
WIP: Adding TA-Lib

### PR DESCRIPTION
**Pull request by masdeseiscaracteres**
https://github.com/mrjbq7/ta-lib

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ environment:
 
   matrix:
     - PLATFORM: "64"
+    - PLATFORM: "32"
```
Comment (mariusvniekerk) [inline .appveyor.yml]: conda-forge no longer supports win32 builds.  please remove this

File: recipes/example/meta.yaml [hunk 2]
```diff
@@ -1,78 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (mariusvniekerk) [inline recipes/example/meta.yaml]: don't delete the example please.  its used for future pull requests.

File: recipes/ta-lib/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  fn: {{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
```
Comment (mariusvniekerk) [inline recipes/ta-lib/meta.yaml]: Add the conda-forge compiler packages (docs (here)[https://conda-forge.org/docs/meta.html#compilers]) please.
You no longer need the toolchain package

File: recipes/ta-lib/bld.bat [hunk 4]
```diff
@@ -0,0 +1,35 @@
+rmdir /s /q "C:\ta-lib\"
+powershell -Command "(New-Object Net.WebClient).DownloadFile('http://prdownloads.sourceforge.net/ta-lib/ta-lib-0.4.0-msvc.zip', 'ta-lib-0.4.0-msvc.zip')"
```
Comment (mariusvniekerk) [inline recipes/ta-lib/bld.bat]: Can you split up the lib package into a separate package.  This seems to be both the python source and and the underlying c library (you can just add another recipe to this PR)
Naming conventions for the package would be as follows.

- ta-lib : the sourceforge c library
- ta-lib-python  : the cython python bindings

Comment (masdeseiscaracteres) [reply recipes/ta-lib/bld.bat]: If creating both at the same time I won't be able to refer to `ta-lib` from `ta-lib-python`, right?

Comment (mariusvniekerk) [reply recipes/ta-lib/bld.bat]: the staged-recipes toolchain is smart enough to make it work :D

File: recipes/ta-lib/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    - {{ compiler('cxx') }}
+  run: []
+
+test:
+  commands:
+    # run ta_regtest executable
+    - ta_regtest
```
Comment (mariusvniekerk) [inline recipes/ta-lib/meta.yaml]: You want to add a `conda inspect linkages {{ name }}  # [not win]` here to see that things are being built as you expect them to be.

**Comment by Peque** [comment]
@masdeseiscaracteres Would you spend some time on this? It seems there are not many changes left to make.

Otherwise, I may open a PR based on yours to push this forward. :blush:

**Pull request closed without merging**